index.js

```diff
@@ -136,6 +139,11 @@ class SlackNotifier extends NotificationBase {
             buildData.settings.slack.statuses = DEFAULT_STATUSES;
         }
 
+        // Add for fixed notification
```

Comment:
Is it nicer?
```js
if (buildData.isFixed) {
    buildData.settings.slack.statuses.push('FIXED');
}

let notificationStatus = buildData.status;

if (buildData.settings.slack.statuses.includes('FAILURE')) {
    if (notificationStatus === 'SUCCESS' && buildData.isFixed) {
        notificationStatus = 'FIXED';
    }
}

if (!buildData.settings.slack.statuses.includes(notificationStatus)) {
    return;
}
```

Reply:
That looks good!
We'll modify it that way.

Reply:
Thank you for fixing. Is there any problem on updating with `buildData.settings.slack.statuses.push('FIXED');` for multiple plugins? I think it is OK if it does not update DB.

Reply:
I think it's okay because it's only slack that's set up.
`buildData.settings.[slack].statuses`

For example, for the email plugin, this is set as follows
`buildData.settings.[email].statuses`

There is no process to update the DB in each notification plugin, so I think okay.